doc/GIT.md

```diff
@@ -111,6 +111,9 @@ or if just the pull request should be checked:
 
 	.*build\W+allow.*
 
+This is only necessry for legady build jobs (that are not defined via
```

Comment:
Typos: 

- `necessry` → `necessary`
- `legady` → `legacy`

Reply:
Thanks, corrected in d8e37a1.